yt_dlp/extractor/bilibili.py

```diff
@@ -69,6 +74,7 @@ def extract_formats(self, play_info):
             'height': int_or_none(video.get('height')),
             'vcodec': video.get('codecs'),
             'acodec': 'none' if audios else None,
+            'dynamic_range': id_to_dr(video.get('id')) or 'SDR',
```

Comment:
```suggestion
            'dynamic_range': {126: 'DV', 125: 'HDR10'}.get(video.get('id')),
```

Reply:
Should I use `SDR` by default? 126 and 125 are the only HDR formats as of now.

Reply:
the default already is `SDR` if the extractor returns `None`, so this is fine as-is